<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>682</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblDescriptionFei">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Punktzahl Teil 1 der Abschlussprüfung</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionFeii">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Punktzahl Teil 2 der Abschlussprüfung</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionFeiiProductplanning">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Planen eines Softwareproduktes</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionFeiiAlgorithmdevelopment">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Entwicklung und Umsetzung von 
Algorithmen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionFeiiEconomy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wirtschafts- und Sozialkunde</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionExaminationprojectPoints">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Planen und Umsetzen eines Softwareprojektes</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionExaminationprojectDefence">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fachgespräch Punktzahl</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescriptionFeiiExaminationproject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Abschlussprojekt</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercentageFei">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>20%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercentageFeiiPp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercentageFeiiAd">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercentageFeiiE">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercentageiEp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>108</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblLanguageOutput">
       <property name="text">
        <string>Sprachausgabe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbLanguageGerman">
       <property name="text">
        <string>Deutsch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbLanguageEnglish">
       <property name="text">
        <string>English</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblGrade">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblGradeFei">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGradeFeiiProductplanning">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGradeFeiiProductplanningAlgorithmdevelopmen">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGradeFeiiProductplanningEconomy">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGradeExaminationprojecti">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeFei">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeFeiiProductplanning_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeFeiiAlgorithmdevelopment_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeFeiieconomy_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeExaminationproject_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeExaminationprojectDefence_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>461</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblTotalresult">
       <property name="text">
        <string> Gesamtergebnis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTotalResult">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblpassed">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:24pt;&quot;&gt;IHK Notenrechner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalc">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>460</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Berechnen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
